addons/crm/tests/test_crm_lead_assignment.py

```diff
@@ -459,6 +459,36 @@ def test_assign_quota(self):
             "Assignment quota: no compensation and no limit anymore (do as asked even with 30 already assigned)"
         )
 
+    # test to mitigate random.choice from raising the zero weight validation error
+    def test_random_choice(self):
+        # create fourth sales team with one member
+        sales_team_4 = self.env['crm.team'].create({
+            'name': 'Sales Team 4',
+            'sequence': 15,
+            'alias_name': False,
+            'use_leads': True,
+            'use_opportunities': True,
+            'company_id': False,
+            'user_id': False,
+            'assignment_domain': [('country_id', '!=', False)],
```

Comment:
Lots of stuff here is not relevant, sometimes even already set implicitly. Just set the fields you need. for the test.

Reply:
Done!